onnx/onnx.in.proto

```diff
@@ -296,6 +296,10 @@ message TensorProto {
     UINT64 = 13;
     COMPLEX64 = 14;     // complex with float32 real and imaginary components
     COMPLEX128 = 15;    // complex with float64 real and imaginary components
+
+    // IEEE single-precision floating-point number truncated to 16 bits.
+    // This format has 1 sign bit, 8 exponent bits, and 7 mantissa bits.
+    BFLOAT16 = 16;
```

Comment:
It would be helpful to add a comment/description to FLOAT16 also to clarify the distinction between that and BFLOAT16.